__tests__/no-deprecated-colors.js

```diff
@@ -165,23 +165,45 @@ testRule({
         true,
         {
           deprecatedFile: path.join(__dirname, '../plugins/lib/primitives-v8.json'),
-          inlineFallback: false,
-        },
+          inlineFallback: false
+        }
       ],
       warnings: [
         {
           message:
             'Variable --color-neutral-emphasis is deprecated for property border. Please use the replacement --borderColor-neutral-emphasis. (primer/no-deprecated-colors)',
           line: 1,
-          column: 6,
+          column: 6
         },
         {
           message:
             'Variable --color-neutral-emphasis is deprecated for property background-color. Please use the replacement --bgColor-neutral-emphasis. (primer/no-deprecated-colors)',
           line: 1,
-          column: 62,
-        },
-      ],
-    },
+          column: 62
+        }
+      ]
+    }
+  ]
+})
+// eslint-disable-next-line no-undef
+testRule({
```

Comment:
shouldn't these be combined?

Reply:
I want to disable the fixable part but maybe it doesn't matter 😅 see what I mean, I've made a huge mess